mne/tests/test_epochs.py

```diff
@@ -145,6 +145,19 @@ def test_epoch_combine_ids():
         # should probably add test + functionality for non-replacement XXX
 
 
+def test_epoch_hierarchical_ids():
+    """Test multi-index/hierarchical indexing
+    """
+    raw, events, picks = _get_data()
+    for preload in [False]:
+        epochs = Epochs(raw, events, {'a/b/a': 1, 'a/b/b': 2, 'a/c': 3,
```

Comment:
Probably won't happen in the real world though ... Just checking if it passes the test.